airbyte-integrations/connectors/destination-iceberg/src/main/java/io/airbyte/integrations/destination/iceberg/IcebergConsumer.java

```diff
@@ -93,10 +93,8 @@ protected void startTracked() throws Exception {
         throw new IllegalStateException("Undefined destination sync mode");
       }
       final boolean isAppendMode = syncMode != DestinationSyncMode.OVERWRITE;
-      AirbyteStreamNameNamespacePair nameNamespacePair = AirbyteStreamNameNamespacePair.fromAirbyteSteam(stream.getStream());
```

Comment:
the integration tests fail for me when removing this line and L99. I think the fix should only be updating the import to `io.airbyte.protocol.models.v0.AirbyteStreamNameNamespacePair` and then updating this line to `AirbyteStreamNameNamespacePair nameNamespacePair = AirbyteStreamNameNamespacePair.fromAirbyteStream(stream.getStream());` (as there's a typo in `Steam`) in order to fix compilation. 

In #23201 I have https://github.com/airbytehq/airbyte/pull/23201/commits/567bc2cb79496d3c3df12c48a87fd5ed2c1e7839 which shows this